docs/libcurl/symbols-in-versions

```diff
@@ -700,7 +700,7 @@ CURLSSH_AUTH_KEYBOARD           7.16.1
 CURLSSH_AUTH_NONE               7.16.1
 CURLSSH_AUTH_PASSWORD           7.16.1
 CURLSSH_AUTH_PUBLICKEY          7.16.1
-CURLSSLBACKEND_AXTLS            7.38.0
+CURLSSLBACKEND_AXTLS            7.38.0       7.63.0
```

Comment:
Just realized that this is actually not quite right, it should have `7.61.0` in the "deprecated" column which is left blank.